app/components/story-tease/template.hbs

```diff
@@ -1,14 +1,32 @@
 {{#if isFancyFeatured}}
-  <div class="box-mat" style='background-image:url({{image-template item.imageMain.template 0 400 item.imageMain.crop}});'></div>
+  <div class="box-mat {{if fullScreen 'box-mat--fullscreen'}}" style='background-image:url({{image-template item.imageMain.template 0 400 item.imageMain.crop}});'></div>
+{{/if}}
+
+{{#if hasBlock}}
+<div class="story-tease-floating-header">
+  {{yield}}
+</div>
 {{/if}}
 
 {{#if isFeatured}}
-  <div class="text text--small text--upper text--heavy text--spacing nudge {{if isFancyFeatured 'text--lightgray'}}">
+  <div class="text text--small text--upper text--heavy text--spacing nudge {{if isFancyFeatured 'text--white'}}">
     <p>{{status}}</p>
   </div>
 {{/if}}
 
-<div class="flag flag--rev flush">
+<div class="story-tease-meta flag {{if flipped 'story-tease--flipped flag--med'}} {{if (not flipped) 'flag--rev'}} flush">
```

Comment:
this class conditional can be rewritten as `{{if flipped 'story-tease--flipped flag--med' 'fleg--rev'}}`